### ISSUE TITLE
`total_height` is incorrect / `Height` tab is missing

### ISSUE THREAD

**Issue by CoryCharlton** [OllisGit/OctoPrint-DisplayLayerProgress#272]
I've been trying to figure out how to fix the `total_height` value which is currently always off by 0.20mm (even was printing at 0.12mm layer height).

The README.md says:

> The total height "calculation" can be done in three ways: 1)the max Z-Value in the G-Code, 2) max Z-Value with extrusion in this height, 3) define a expression like this ;MAXZ:([0-9]+[.]*[0-9]*).* to evaluate the max height https://github.com/OllisGit/OctoPrint-DisplayLayerProgress/issues/82

But I couldn't actually track down _where_ to set that value. Digging through the commits I see that the expression was added in version [`1.13.0`](https://github.com/OllisGit/OctoPrint-DisplayLayerProgress/commit/dcd8ff36ef21b6eb1e4082e384953a63a14b903f) and should be configurable on the `Height` tab. 

The problem is that it looks like the `Height` tab was removed in version `1.23.0` and it's not clear why.

Here's an example of the problem where the total height shows as 12.20mm but I'm printing at 0.12mm layer height so that value shouldn't technically be possible.

![MedicalMistySunfish](https://user-images.githubusercontent.com/10094287/216472430-fcc72044-cc78-42d4-b58e-f6829d6ed4b8.png)

I'll pull down the code later to see if I can track down the root cause.

Edit: I am running the latest RC: 1.29.0rc1

**Comment by CoryCharlton** [comment]
Here's the GCode from my example screenshot: [CE3E3V2_Valentines 2023.zip](https://github.com/OllisGit/OctoPrint-DisplayLayerProgress/files/10574227/CE3E3V2_Valentines.2023.zip)

**Comment by CoryCharlton** [comment]
It looks like the height is coming from the OctoPrint metadata which is incorrect so this likely isn't your issue. Looks like I'll be pulling down the OctoPrint code next :D

```
"dimensions": {
  "width": 158.56700744479895,
  "depth": 180.0,
  "height": 12.199999809265137
},
```

**Comment by CoryCharlton** [comment]
This has been confirmed to be an OctoPrint issue and will be resolved in the 1.9.0 release. https://github.com/OctoPrint/OctoPrint/issues/4724#issuecomment-1416681127
